include/nbl/asset/filters/CBlitImageFilter.h

```diff
@@ -614,6 +743,14 @@ class CBlitImageFilter : public CImageFilter<CBlitImageFilter<Swizzle,Dither,Nor
 			// obviously we have multiple channels and each channel has a certain type for arithmetic
 			return texelCount*MaxChannels*sizeof(value_type);
 		}
+
+		static inline core::vectorSIMDu32 getPhaseCount(const core::vectorSIMDu32& in, const core::vectorSIMDu32& out, const asset::IImage::E_TYPE inImageType)
+		{
+			core::vectorSIMDu32 result(0u);
```

Comment:
initialize to 1 maybe?

Reply:
then an assert that none of the phases is 0 is not troublesome

Reply:
On the contrary, a default value of 1 would be troublesome. It would make [this size calculation](https://github.com/Devsh-Graphics-Programming/Nabla/blob/fd9be09ea1b0770f439554a1e2fd4035a7049e42/include/nbl/asset/filters/CBlitImageFilter.h#L294) incorrect. If we put a check there to see if `phaseCount == 1`, then to know if its truly a monophase case or just a default value for a non-existent axis, we would need more checks.

Reply:
Related: https://github.com/Devsh-Graphics-Programming/Nabla/pull/140#discussion_r863613193

Reply:
ok